src/api-sdks/connection-api/clients/csharp/src/IdeaStatiCa.ConnectionApi/ConnectionApiServiceAttacher.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Threading.Tasks;
+
+namespace IdeaStatiCa.ConnectionApi
+{
+	public class ConnectionApiServiceAttacher : IApiServiceFactory<IConnectionApiClient>
+	{
+		string BaseUrl { get; set; }
+
+		/// <summary>
+		/// Con
+		/// </summary>
+		/// <param name="baseUrl"></param>
+		public ConnectionApiServiceAttacher(string baseUrl)
+		{
+			this.BaseUrl = baseUrl;
+		}
+
+		/// <inheritdoc cref="IApiServiceFactory.CreateConnectionApiClient"/>
+		public async Task<IConnectionApiClient> CreateApiClient()
+		{
+			var client = new ConnectionApiClient(BaseUrl);
+			await client.CreateAsync();
+			return await Task.FromResult(client);
```

Comment:
since there is already await on line above, is it not enought to just return client ?

Reply:
Agree